spec/support/controllers_helper.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+RSpec.shared_context 'shared controller data', shared_context: :metadata do
```

Comment:
so normally helper should look like module and after that we include it rails_helper.rb
like:
  config.include Devise::Test::ControllerHelpers, type: :controller